### DISCUSSION TITLE
想要增加第五章的话，下面的操作对吗

### DISCUSSION THREAD

**Discussion by Sillydog2333** [whutug/whu-thesis#112] [Q&A]
1. 新建chapter5.tex和chapter5.aux文件
2. 将前面4章中的aux文件内容复制到chapter5中（前四章的aux文件内容看起来相同，对吗？）
3. 在whu-thesis-demo.tex中加入\include{pages/chapter5}

**Comment by xkwxdyy** [comment]
`.aux` 相关的全都不要

**Reply by xkwxdyy** [reply]
1. 在 `pages` 新建 `chapter5.tex`
2. 在 `whu-thesis-demo.tex` 中加入 `\include{pages/chapter5.tex}`

**Reply by Sillydog2333** [reply]
好的，非常感谢！

**Reply by Sillydog2333** [reply]
太及时了，感谢感谢！
